### PR TITLE
Feature/156 Update folder structure component types

### PR DESCRIPTION
**Pull request by BeneRichi**
## Pull request
Created new directories for element and layout .vue components to have a more structured folder tree. To make them accessible and installable for Vue, I changed the @import urls of those components in component.js and styleguide.routes.js.
 
### Ticket
[156](https://github.com/valantic/vue-template/projects/1#card-87480305)
 
### Browser testing
- [*link*](http://localhost:9090/styleguide/sandbox/layout)
 
### Checklist
- [x] I merged the current development branch (before testing)
- [ ] Added JSDoc and styleguide demo
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [ ] Did run automated tests and linters
- [x] Did assign ticket
- [x] Double checked target branch
 
## Review/Test checklist
- [x] Did review code and documentation
- [x] Tested all links in project relevant browsers
- [ ] Tested all links in different screen sizes
- [ ] Did check accessibility (Wave, only errors)
- [x] Re-assign ticket to developer

### CODE REVIEW

**Review by patric-eberle — CHANGES_REQUESTED** [review]
Good work. But we should invest a bit more time here:

todo:
- [ ] Move all imports (except for the layout component) to the single components.
- [ ] We should also apply the eslint config to the new folders. Please connect with me if you get here, because there is a better way then the local config file.

File: src/setup/components-global.js [hunk 1]
```diff
@@ -1,21 +1,12 @@
-// Form elements
```
Comment (patric-eberle) [inline src/setup/components-global.js:1]: todo: I would prefer if we could name the file `ssr-components.js` instead. (SSR = Server Side Rendering)

File: src/setup/components-global.js [hunk 2]
```diff
@@ -1,21 +1,12 @@
...
-import eSelect from '@/elements/e-select.vue';
-import eRadio from '@/elements/e-radio.vue';
-import eCheckbox from '@/elements/e-checkbox.vue';
-import eTextarea from '@/elements/e-textarea.vue';
-import eButton from '@/elements/e-button.vue';
-
 // Elements
 import eIcon from '@/elements/e-icon.vue';
 import ePicture from '@/elements/e-picture.vue';
 import eProgress from '@/elements/e-progress.vue';
```
Comment (patric-eberle) [inline src/setup/components-global.js:4]: todo: Please remove these as well.

File: src/setup/components-global.js [hunk 3]
```diff
@@ -48,13 +32,6 @@ export default {
       });
     } else {
```
Comment (patric-eberle) [inline src/setup/components-global.js:33]: todo: the `else` can be removed. it was only added for PhpStorm so it knows, which components are already imported on global scale. Since we only import components that are most likely not imported by other components we can drop this.

**Review by patric-eberle — CHANGES_REQUESTED** [review]
Browser review

- [ ] On http://localhost:9090/styleguide/sandbox/notifications the button component is missing.
- [ ] Please resolve the merge conflicts shown at the bottom of the PR. You can merge the target branch on your local machine to see the conflicts and resolve them. In a few edge cases your local git will be able to resolve certain conflicts automatically.

Please make sure you always try to test code changes in the browser. This is not possible in the template for all changes unfortunately, but you could still check the pages we already have (checking for console errors).

**Comment by BeneRichi** [comment]
@patric-eberle Tickets and merge have been resolved. Please review again.
